internal/util/rocksmq/server/rocksmq/rocksmq_impl.go

```diff
@@ -415,10 +456,14 @@ func (rmq *rocksmq) RegisterConsumer(consumer *Consumer) {
 		rmq.consumers.Store(consumer.Topic, consumers)
 	}
 	log.Debug("Rocksmq register consumer successfully ", zap.String("topic", consumer.Topic), zap.Int64("elapsed", time.Since(start).Milliseconds()))
+	return nil
 }
 
 // DestroyConsumerGroup removes a consumer group from rocksdb_kv
 func (rmq *rocksmq) DestroyConsumerGroup(topicName, groupName string) error {
+	if rmq.isClosed() {
```

Comment:
In Close(), rmq will be set to closed firstly, so DestroyConsumerGroup will be returned here.

Reply:
fix by adding a internal call of DestroyConsumerGroup